### PR TITLE
fixing the 0/0 = infinity theory

### PR DESCRIPTION
**Pull request by Volas171**
idk someone on yeet hosting keeps saying 0/0 = infinity due to this
image
soooo
I created a "fix" for it (needs review)
its clearly not a good fix as it can be bypassed but its something to stop this 
image

### CODE REVIEW

**Review by Felix-44 — COMMENTED** [review]
This is an interesting hack
but..you do know &eval is gonna be replaced by a safe &debug, right?

**Comment by Volas171** [comment]
> This is an interesting hack but..you do know &eval is gonna be replaced by a safe &debug, right?

understood, but it can be used with the debug anyways as it is just checking if there's 0/(number) or number/0 and returns the output skipping actual eval executing.

**Comment by Felix-44** [comment]
> > This is an interesting hack but..you do know &eval is gonna be replaced by a safe &debug, right?
> 
> understood, but it can be used with the debug anyways as it is just checking if there's 0/(number) or number/0 in message content and returns the output skipping actual eval executing.
> 
> **TL;DR you just need to change the "eval" from it and copy the code** (lol)

guess we can merge it then

**Comment by LJMSU** [comment]
0/0 != Infinity logically or mathematically
